ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
The client tools should just depend on the client code from the ecm files they use being built.  I'd have thought it was covered by "espscm" which should build all of the ecm files?